### ISSUE TITLE
Graph Becomes Distorted

### ISSUE THREAD

**Issue by Mioshek** [aqua30/GraphCompose#3]
Device Xiaomi Poco F2 Pro
Android Version 12 (Api Level 31)
<p>After changing activity to 
    <ol>
        <li>OnPause</li>
        <li>OnStop</li>
        <li>OnStart</li>
        <li>OnResume</li>
    </ol>
    Graph becomes distorted.
</p>
<img src="https://github.com/aqua30/GraphCompose/assets/90716960/b2bcc86f-60a6-4e2d-852b-32e28b648e6e">

<img src="https://github.com/aqua30/GraphCompose/assets/90716960/7c2ed0af-adbf-4643-9978-7dc557858fe3">

**Comment by WilliamBankin** [comment]
Hello, was a solution found?

**Comment by Mioshek** [comment]
No, but I made my own graph generation code here https://github.com/Mioshek/ChartPlanner/blob/main/app/src/main/java/com/mioshek/chartplanner/views/charts/Graph.kt
Feel free to use it and modify.
